angular.json

```
@@ -76,86 +81,46 @@
                         "stylePreprocessorOptions": {
                             "includePaths": ["dist", "apps/docs/src"]
                         },
-                        "styles": [
-                            "apps/docs/src/main.scss"
-                        ],
+                        "styles": ["apps/docs/src/main.scss"],
                         "scripts": [],
-                        "vendorChunk": true,
-                        "extractLicenses": false,
-                        "buildOptimizer": false,
-                        "optimization": false,
                         "namedChunks": true,
-                        "allowedCommonJsDependencies": [
-                            "highlight.js",
-                            "@messageformat/core",
-                            "path-normalize"
-                        ]
+                        "browser": "apps/docs/src/main.ts",
+                        "server": "apps/docs/src/main.server.ts",
+                        "prerender": true,
+                        "ssr": {
+                            "entry": "apps/docs/src/server.ts"
+                        }
```

Нам же по сути SSG (пререндера) достаточно, может отключим ssr?

```suggestion
                        "ssr": false
```

---

Оставь будет как есть, определимся что нам нужно и если надо будет, то переделаем